packages/workbox-core/_private/cacheWrapper.mjs

```diff
@@ -64,10 +64,10 @@ const putWrapper = async ({
   }
 
   if (process.env.NODE_ENV !== 'production') {
-    if (responseToCache.method && responseToCache.method !== 'GET') {
+    if (request.method && request.method !== 'GET') {
```

Comment:
Good catch. My assumption is that this is not a breaking change, as the `cache.put()` later on would have rejected if the `request`'s `mode` wasn't already `'GET'`.